spec/index.bs

```diff
@@ -2731,6 +2748,8 @@ interface GPUAdapter {
                     Otherwise:
 
                     1. Let |device| be [=a new device=] with the capabilities described by |descriptor|.
+                        - Any limit associated with a disabled feature is set to `undefined`.
+                        - Any unrequested limit associated with an enabled feature is set to its default.
```

Comment:
These would maybe benefit from doing a for loop over the values like is done several times above?